src/js/containers/awardV2/visualization/AwardAmountsContainer.jsx

```diff
@@ -99,16 +100,21 @@ export class AwardAmountsContainer extends React.Component {
         });
 
         // Store the counts in Redux for use in the referenced awards table
+        // and related awards section
         this.props.setCounts({
             idvs: data.idv_count,
-            contracts: data.contract_count
+            contracts: data.contract_count,
+            total: data.idv_count + data.contract_count
         });
```

Comment:
Looks like we're putting into redux an object with a different shape than what exists in the model.

Is that true for what we currently have prior to the delivery of this PR?

Award Model Object !== `redux.award`?